backend/internal/storage/database.go

```diff
@@ -0,0 +1,55 @@
+package storage
+
+import (
+	"fmt"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+// Configuration to connect to database
+type DsnConfig struct {
+	// Username to login database
+	User string
+	// User password to verify user
+	Password string
+	// Database name to be accessed
+	DatabaseName string
+	// The host used to host database
+	Host string
+	// The port used to connect database
```

Comment:
"The port used to connect database" -> "The port used to connect to the database"